### DISCUSSION TITLE
Pivot Grid Values on Rows?

### DISCUSSION THREAD

**Discussion by GrimnebulinAhab** [TabularEditor/TabularEditor3#974] [🙋♀️TE3 Q&A] [answered]
Just curious if there's a way to do this in the pivot grid in the same way as the matrix visual in PBI. Sorry if it's available and I'm just not seeing it. Thanks in advance for your help!

**Comment by otykier** [comment, marked as answer]
Hi @GrimnebulinAhab 
Apologies for the late reply. This is possible by dragging the "Values" box to the column area, as demonstrated in the gif below:

![pivotgrid](https://github.com/TabularEditor/TabularEditor3/assets/8976200/e083268f-2318-41c4-a51b-52a2b4a5ac84)

**Reply by GrimnebulinAhab** [reply]
Wonderful. Thank you!
